clang/lib/Frontend/VerifyDiagnosticConsumer.cpp

```diff
@@ -1098,7 +1098,13 @@ void VerifyDiagnosticConsumer::CheckDiagnostics() {
     // Produce an error if no expected-* directives could be found in the
     // source file(s) processed.
     if (Status == HasNoDirectives) {
-      Diags.Report(diag::err_verify_no_directives).setForceEmit();
+      std::string directives;
+      if (Diags.getDiagnosticOptions().VerifyPrefixes.empty()) {
+        directives = "expected";
```

Comment:
I don't think this can be empty. Have you checked that this can actually happen?

Reply:
yes, when there are no diagnostics passed in the verify, ie. when we pass ```-verify=``` only. In this case ```expected-no-directives``` should be the error output when no directives are present. and when we pass ```-verify=foo``` then ```foo-no-diagnostics``` should be the error when no foo directives are present. If I don't include this check, then with only ```-verify=``` it results in a segmentation fault.

Reply:
any thoughts on this?

Reply:
I guess that's fine then.